bin/start-dashboard.sh

```diff
@@ -26,16 +26,14 @@ load_rss_env
 
 cd "$RSS_HOME"
 
-COORDINATOR_CONF_FILE="${RSS_CONF_DIR}/coordinator.conf"
+DASHBOARD_CONF_FILE="${RSS_CONF_DIR}/dashboard.conf"
```

Comment:
This will add a dashboard.conf configuration file, and the documentation needs to be modified.